official/resnet/keras/keras_common.py

```diff
@@ -252,6 +252,11 @@ def input_fn(is_training, data_dir, batch_size, *args, **kwargs):
   return input_fn
 
 
+def is_v2_0():
+  """Returns true if using tf 2.0."""
```

Comment:
This seems fine to me. cc @annarev in case she knows better ways to check for 2.0 or not that work in both 1.x and 2.0

Reply:
Not having contrib is a cheat we used in distribution_utils.  hasattr(tf, 'contrib')  Yuefeng thought of that one.